core/src/apps/management/recovery_device/layout.py

```diff
@@ -45,8 +44,20 @@ async def request_mnemonic(
     from . import word_validity
     from trezor.ui.layouts.common import button_request
     from trezor.ui.layouts.recovery import request_word
+    from trezor import utils
+
+    if utils.MODEL in ("R",):
+        await confirm_action(
+            ctx,
+            "request_word",
+            "WORD ENTERING",
+            description="You'll only have to select the first 2-3 letters.",
```

Comment:
But this is how it actually works, right? That you don't need to know the whole words.
Some users were afraid they would need to type in the whole words. By that we wanted to avoid that feeling/frustration/fear…

Reply:
in that case we should say "4 characters", not 2 to 3

Reply:
For majority of words 3 is enough. For some maybe even 2?

But let's review it later on. I don't want this to slow us.